core/domain/opportunity_services.py

```diff
@@ -56,12 +56,15 @@ def is_exploration_available_for_contribution(exp_id):
     return model is not None
 
 
-def get_exploration_opportunity_summary_from_model(model):
+def get_exploration_opportunity_summary_from_model(
+    model, translations_in_review):
```

Comment:
Instead of modifying this function, can you create a new one? When we have get from model functions they usually accept only the model and not any other arguments.

Reply:
Done!